spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/BinderPropertyKeys.java

```diff
@@ -132,8 +132,13 @@
 	public static final String DURABLE = "durableSubscription";
 
 	/**
-	 * Minimum partition count, if the transport supports partitioning natively (e.g. Kafka)
+	 * Minimum partition count, if the transport supports partitioning natively (e.g. Kafka).
 	 */
 	public static final String MIN_PARTITION_COUNT = "minPartitionCount";
 
+	/**
+	 * Flag to indicate if the Kafka producer needs to be synchronous/asynchronous.
+	 */
```

Comment:
Should we perhaps make it a Kafka property only and set it on the binder directly (with a default option) ? Doesn't feel like a generic property. (minPartitionCount is an unfortunate anomaly, as for durable, we might want to consider making removing it at the top level too).